.ci/ogc/build.sh

```diff
@@ -18,7 +18,6 @@ cmake -GNinja \
  -DWITH_QWTPOLAR=OFF \
  -DWITH_APIDOC=OFF \
  -DWITH_ASTYLE=OFF \
- -DWITH_GEOREFERENCER=OFF \
```

Comment:
Should be replaced with `WITH_GSL=OFF` I think

Reply:
@m-kuhn , we could test that yeah. It builds fine with GSL, but not a bad idea to test the WITH_GSL=OFF flag here.